src/hash_table.py

```diff
@@ -0,0 +1,39 @@
+"""Hash table implementation."""
+
+
+class HashTable(object):
+    """Hash Table."""
+
+    def __init__(self):
```

Comment:
"The number of slots in the table should be determined when the table is initialized, by passing an argument: `foo = HashTable(1024)`"